api/app/src/external/commonwell/link/create.ts

```diff
@@ -71,26 +74,20 @@ export const create = async (
       lola1Links.forEach(async link => {
         if (link._links?.upgrade?.href) {
           requests.push(
-            commonWell.upgradeOrDowngradeNetworkLink(queryMeta, link._links.upgrade.href)
+            commonWell
+              .upgradeOrDowngradeNetworkLink(queryMeta, link._links.upgrade.href)
+              .catch(err => {
+                console.log(`Failed to upgrade link: `, err);
+                Sentry.captureException(err, { extra: { cxId, cwPatientId, personId, context } });
+                throw err;
```

Comment:
general comment for all the places we're handling CW errors: can pushing the raw err leak sensitive info?

Reply:
It shouldn't, and Sentry has local and server-side PII filter: https://docs.sentry.io/platforms/node/data-management/sensitive-data/

I suggest we push this and run in staging so we see what will be available there.